.github/workflows/ci.yaml

```diff
@@ -88,7 +88,9 @@ jobs:
             if [[ ${{ matrix.os }} == "macos-latest" ]]; then
               export PYOPENCL_TEST=port:pthread
             fi
-            python -m pytest --durations=10 --tb=native --junitxml=pytest.xml --doctest-modules -rxsw .
+            python -m pytest --durations=10 --tb=native --junitxml=pytest.xml --doctest-modules -rxsw ./euler
+            python -m pytest --durations=10 --tb=native --junitxml=pytest.xml --doctest-modules -rxsw ./wave
+            python -m pytest --durations=10 --tb=native --junitxml=pytest.xml --doctest-modules -rxsw ./diffusion
```

Comment:
Add a comment linking back to the issue to explain why this is being done.

Reply:
(65be23a)